controller/controller.go

```diff
@@ -272,6 +349,14 @@ func deleteSchedulerHelper(logger logrus.FieldLogger, mr *models.MixedMetricsRep
 		}
 	}
 
+	err = mr.WithSegment(models.SegmentDelete, func() error {
```

Comment:
It does, the problem was calling CreateNamespaceIfNecessary before GetSchedulerScalingInfo